test/extensions/quic_listeners/quiche/platform/quic_platform_test.cc

```diff
@@ -350,6 +355,11 @@ TEST_F(QuicPlatformTest, QuicLog) {
   EXPECT_EQ(3, i);
 }
 
+TEST_F(QuicPlatformTest, QuicLogVector) {
+  std::vector<int32_t> v = {1, 2, 3, 4, 5};
+  EXPECT_LOG_CONTAINS("info", "vector { 1, 2, 3, 4, 5 }", QUIC_LOG(INFO) << v);
```

Comment:
Does it work? IIRC, the default log level is ERROR, and INFO will be logged unless you do `GetLogger().set_level(INFO)`

Reply:
It shouldn't. But it did pass. It seems to be a bug in quic logging implementation. Anyway, the purpose of this test is testing operator<< overload, not logging level. I changed it to output at error instead.

Reply:
It works because the macro forcefully set the log level to "trace".